### ISSUE TITLE
Can't connect to the RPC interface

### ISSUE THREAD

**Issue by ernst-at-tv2** [severalnines/docker#25]
When inside the container we can easily curl localhost:9500… but if we expose the port to  0.0.0.0:9500 on the host, then can telnet to the port but the connection is immediately closed.

How can we expose the RPC interface so we can call it remotely? or just from the docker host.

**Comment by kedazo** [comment]
Hey, just add the following line to /etc/default/cmon inside the container
( or here : https://github.com/severalnines/docker/blob/master/conf/cmon.default )

RPC_BIND_ADDRESSES=0.0.0.0

and restart the cmon/container. You know the cmon by default only binds to localhost.

**Comment by ernst-at-tv2** [comment]
@kedazo sorry forgot to mention that we've tried doing that by changing the config to `RPC_BIND_ADDRESSES="127.0.0.1,0.0.0.0"` and doing `supervisorctl restart cmon`

we can connect to port 9500 from the host but the connections is closed instantly

from inside the container:
```shell
jeer@a:~$ docker exec -ti clustercontrol_controller bash
[root@a /]# cat /etc/default/cmon
EVENTS_CLIENT="http://127.0.0.1:9510"
RPC_BIND_ADDRESSES="127.0.0.1,0.0.0.0"
CLOUD_SERVICE="http://127.0.0.1:9518"
[root@a /]# curl localhost:9500
{
    "cc_timestamp": 1581939971,
    "errorString": "Path is invalid ('/').",
    "requestStatus": "InvalidRequest"
}
```

but trying from the docker host:
```
jeer@a:~$ curl localhost:9500
curl: (56) Recv failure: Connection reset by peer
```

output of `docker ps`:
```
fd016eb0b79a        tv2test/clustercontrol:1.7.5   "/entrypoint.sh"    4 days ago          Up 17 seconds (health: starting)   0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp, 9501/tcp, 9510-9511/tcp, 0.0.0.0:9500->9500/tcp, 9999/tcp   clustercontrol_controller
```

**Comment by ashraf-s9s** [comment]
Hi @ernst-at-tv2 

The Docker image uses supervisorctl to start cmon in foreground, thus skipping `/etc/default/cmon`. I've pushed a change to this line:
https://github.com/severalnines/docker/blob/master/conf/supervisord.conf#L39

Plus some other improvements in entrypoint script handling. The new image is now available in Docker Hub (tag: latest or tag:1.7.5)
